pom.xml

```diff
@@ -215,6 +228,9 @@
           <execution>
             <id>sign-artifacts</id>
             <phase>verify</phase>
+            <configuration>
+              <skip>true</skip>
```

Comment:
@GrigorievNick Do you mind to enlighten us why you need to set the skip to true?

Reply:
@cheeseng I am sorry I just deploy this to my own artifactory.
and accidentally commit custom pom details.